learningresources/tests/test_views.py

```diff
@@ -68,6 +70,58 @@ def test_listing_unauthorized(self):
         self.assertTrue(resp.status_code == UNAUTHORIZED)
         self.assertTrue("unauthorized" in body)
 
+    def test_listing_importcourse_perms(self):
+        """
```

Comment:
This test might be easier to follow with a similar function to the one mentioned above. Maybe even make it part of the class.